packages/core/src/messages/responses/cache-get.ts

```diff
@@ -29,14 +30,30 @@ const TEXT_DECODER = new TextDecoder();
  * }
  * ```
  */
-export abstract class Response extends ResponseBase {}
+export abstract class Response extends ResponseBase {
+  public value(): string | undefined {
+    if (this instanceof CacheGet.Hit) {
```

Comment:
Does this need to import CacheGet.Hit instead of referencing the class in this file?

Reply:
agreed will fix

Reply:
addressed in https://github.com/momentohq/client-sdk-javascript/pull/760/commits/383af5f8b5f34a70ce4a0cf6efa6bd440dd70fb6